src/overlays/misc/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -6,35 +6,162 @@
 #include "gfx_setupdl.h"
 #include "gfxalloc.h"
 #include "printf.h"
+#include "translation.h"
 #include "z64play.h"
 #include "z64save.h"
 
 #include "assets/textures/parameter_static/parameter_static.h"
 
+typedef enum DebugSection {
+    /* 0x00 */ SECTION_RUPEES,
+    /* 0x01 */ SECTION_HEALTH_CAPACITY,
+    /* 0x02 */ SECTION_CURRENT_HEALTH,
+    /* 0x03 */ SECTION_FIRST_INVENTORY_SLOT,
+    /* 0x1A */ SECTION_LAST_INVENTORY_SLOT = 0x1A,
+    /* 0x1B */ SECTION_FIRST_KEY_COUNT,
+    /* 0x2B */ SECTION_LAST_KEY_COUNT = 0x2B,
```

Comment:
ngl an enum for something like this doesnt make too much sense to me. would prefer defines at this point cause of how spread out they are.